charts/alerts/templates/deployment.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-alerts-app
+  labels:
+    app: alerts-app
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: alerts-app
+  template:
+    metadata:
+      annotations:
+      labels:
+        app: alerts-app
+    spec:
+      imagePullSecrets:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: ghcr.io/cdcgov/phdi/alerts:{{ .Values.image.tag }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: {{ .Chart.Name }}
+              containerPort: {{ .Values.service.port }}
+          env:
+            - name: SMARTY_AUTH_ID
```

Comment:
I don't think the alerting service needs the Smarty credentials, but it seems like there are some other env vars that do need to be configurable - `SLACK_BOT_TOKEN`, `TEAMS_WEBHOOK_URL`, and `COMMUNICATION_SERVICE_NAME` I think. I think you'll need to update the chart so any of those can be set as needed by the user.

I got those env var names from the docs btw: https://cdcgov.github.io/phdi/latest/containers/alerts.html

Reply:
I _think_ they'll need to be set up as Secrets, too - seems like the Slack one at least should be an OAuth token.

Reply:
added